recipe/meta.yaml

```diff
@@ -25,28 +25,28 @@ requirements:
     build:
         - python
         - setuptools
+        - cmake  # [win]
         - numpy x.x
         - hdf4  # [not win]
         - hdf5 1.8.15.1
         - geos
         - proj.4
         - xerces-c
-        - libnetcdf
+        - libnetcdf 4.3*
         - krb5
-        - libtiff
-        - libpng
-        - openjpeg
+        - libpng 1.6*
+        - openjpeg  # [unix and py<34 or win]
+        - libtiff 4*  # [unix and py<34 or win]
```

Comment:
Windows should be safe from the `xz` [issue](https://github.com/conda-forge/gdal-feedstock/pull/37/files#r60475113) as we don't link anything to our `xz` on Windows.